packages/mg-imagescraper/lib/ImageScraper.js

```diff
@@ -129,6 +132,28 @@ class ImageScraper {
         return value;
     }
 
+    async processMobiledoc(value) {
+        let json = JSON.parse(value);
+
+        // TODO: Loop over `json` as a whole and find all `src` keys instead of iterating over specific card types
```

Comment:
I _think_ it may only be those two, but wanted to future-proof incase any new cards are added. I've now updated this to recursively loop over the whole object and look for any `src` keys, and download those.